pandas/core/dtypes/common.py

```diff
@@ -168,11 +168,11 @@ def ensure_int_or_float(arr: ArrayLike, copy=False) -> np.array:
     will remain unchanged.
     """
     try:
-        return arr.astype("int64", copy=copy, casting="safe")
+        return arr.astype("int64", copy=copy, casting="safe")  # type: ignore
```

Comment:
what are these complaining about? I really really don't like the type ignore because it is never again checked even if something else is fixed .

Reply:
Check out https://github.com/pandas-dev/pandas/pull/27424#issuecomment-512007383 - I think a problem with ExtensionArrays going through this method where their `astype` signature doesn't align with the other types that come through here

Reply:
ok, so mypy is failing on this then? I agree might be a problem, but hiding it is not the answer (or if you do then pls open an issue and point to that issue so can address elsewhere).